### DISCUSSION TITLE
Can the tree file (Newick format) contain the node Bootstrp check value?

### DISCUSSION THREAD

**Discussion by Jiahe-Sun** [hahnlab/CAFE5#107] [Q&A]
Can the tree file (Newick format) contain the node Bootstrp check value?
As follows：
[9_plant_r8s_ultrametric.txt](https://github.com/hahnlab/CAFE5/files/10146824/9_plant_r8s_ultrametric.txt)

**Comment by benfulton** [comment]
I don't understand what this means.
